configs/milvus.yaml

```diff
@@ -200,132 +205,135 @@ proxy:
-      query: 
+      query:
         format: "[$time_now] [ACCESS] <$user_name: $user_addr> $method_name [status: $method_status] [code: $error_code] [sdk: $sdk_version] [msg: $error_msg] [traceID: $trace_id] [timeCost: $time_cost] [database: $database_name] [collection: $collection_name] [partitions: $partition_name] [expr: $method_expr]"
-          # set formatter owners by method name(method was all milvus external interface)
-          # all method will use base formatter default
-          # one method only could use one formatter
-          # if set a method formatter mutiple times, will use random fomatter.
-        methods: ["Query", "Search", "Delete"]
-    # localPath: /tmp/milvus_accesslog // log file rootpath
-    # maxSize: 64 # max log file size(MB) of singal log file, mean close when time <= 0.
-    # rotatedTime: 0 # max time range of singal log file, mean close when time <= 0;
-    # maxBackups: 8 # num of reserved backups. will rotate and crate a new backup when access log file trigger maxSize or rotatedTime.
-    # cacheSize: 10240 # write cache of accesslog in Byte
-
-    # minioEnable: false # update backups to milvus minio when minioEnable is true.
-    # remotePath: "access_log/" # file path when update backups to minio
-    # remoteMaxTime: 0 # max time range(in Hour) of backups in minio, 0 means close time retention.
+        methods: "Query,Search,Delete"
+  connectionCheckIntervalSeconds: 120 # the interval time(in seconds) for connection manager to scan inactive client info
+  connectionClientInfoTTLSeconds: 86400 # inactive client info TTL duration, in seconds
+  maxConnectionNum: 10000 # the max client info numbers that proxy should manage, avoid too many client infos
+  gracefulStopTimeout: 30 # seconds. force stop node without graceful stop
+  slowQuerySpanInSeconds: 5 # query whose executed time exceeds the `slowQuerySpanInSeconds` can be considered slow, in seconds.
   http:
     enabled: true # Whether to enable the http server
     debug_mode: false # Whether to enable http server debug mode
-  # can specify ip for example
-  # ip: 127.0.0.1
-  ip: # if not specify address, will use the first unicastable address as local ip
+    port:  # high-level restful api
+    acceptTypeAllowInt64: true # high-level restful api, whether http client can deal with int64
+    enablePprof: true # Whether to enable pprof middleware on the metrics port
+  ip:  # if not specified, use the first unicastable address
   port: 19530
   internalPort: 19529
   grpc:
     serverMaxSendSize: 268435456
     serverMaxRecvSize: 67108864
     clientMaxSendSize: 268435456
     clientMaxRecvSize: 67108864
-  # query whose executed time exceeds the `slowQuerySpanInSeconds` can be considered slow, in seconds.
-  slowQuerySpanInSeconds: 5
 
 # Related configuration of queryCoord, used to manage topology and load balancing for the query nodes, and handoff from growing segments to sealed segments.
 queryCoord:
+  taskMergeCap: 1
```

Comment:
Any description or comments here?

Reply:
Will fill the descriptions in future PRs, thanks!